src/Components/UI/Modal/Signup.jsx

```diff
@@ -0,0 +1,126 @@
+import Button from "../Button/Button";
+import "./Signup.css";
+import InputTypeOne from "../Input/InputTypeOne";
+import { useModal } from "../../../Context";
+import { useState } from "react";
+
+const initialSignupState = {
+  firstName: "",
+  lastName: "",
+  email: "",
+  password: "",
+};
+
+const Signup = () => {
+  const { setShowLogin, setShowSignup } = useModal();
+  const [user, setUser] = useState(initialSignupState);
+
+  const onSignupFormSubmitHandler = (e) => {
+    e.preventDefault();
+  };
+
+  const onInputChangeHandler = (e) => {
+    const name = e.target.name;
+    const value = e.target.value;
+
+    setUser((oldData) => {
+      return {
+        ...oldData,
+        [name]: value,
+      };
+    });
+  };
+
+  const onCloseClick = () => {
+    setShowLogin(false);
+    setShowSignup(false);
+  };
+
+  const onLoginClick = () => {
+    setShowLogin(true);
+    setShowSignup(false);
+  };
+
+  return (
+    <>
+      <div
+        className="modal-backdrop"
+        onClick={() => {
+          setShowSignup(false);
+        }}
+      ></div>
+      <div className="signup-modal-one">
+        <h1>Sign Up</h1>
+        <p>Please provide your details.</p>
+        <a onClick={onCloseClick}>
+          <i className="fas fa-times"></i>
+        </a>
```

Comment:
Suggestion: use <button> tag instead of <a> tag as you are specifically using as button.

Reply:
Ok.